### PR TITLE
Added input and slider component

### PR DESCRIPTION
**Pull request by sanjitsarkar**
added styling for input fields and slider.

### CODE REVIEW

File: css/components/alert/index.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
+.alert {
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+  padding: calc(var(--pd) * 0.5) calc(var(--pd) * 1.5);
+  border-radius: calc(var(--br-radius) * 0.5);
+  box-shadow: var(--bx-sh-3);
+  background: var(--primary-color);
+  color: var(--light-color);
+  z-index: 9999;
```
Comment (deekshithmd) [inline css/components/alert/index.css]: don't use such big value for z-index

File: css/components/avatar/index.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
+.avatar {
+  border-radius: 50%;
+  border: 0.3rem solid var(--light-color);
```
Comment (deekshithmd) [inline css/components/avatar/index.css]: use px for border

File: css/components/badge/index.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+}
+.badge {
+  right: -5%;
+  top: -5%;
+  position: absolute;
+  display: grid;
+  place-content: center;
+  border-radius: 50%;
+  width: 1.5rem;
+  padding: 0.25rem;
```
Comment (deekshithmd) [inline css/components/badge/index.css]: use px for values less than 10px

File: css/components/button/index.css [hunk 4]
```diff
@@ -0,0 +1,152 @@
+button,
+input[type="submit"] {
+  cursor: pointer;
+  outline: none;
+  border: none;
+  padding: 0.5rem 0.6rem;
+  border-radius: 0.3rem;
```
Comment (deekshithmd) [inline css/components/button/index.css]: use px for border radius

File: css/components/button/index.css [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+  opacity: 0.9;
+}
+.btn:active {
+  box-shadow: var(--bx-sh-3);
+  opacity: 0.7;
+
+  transform: scale(0.95);
+}
+.btn-disabled {
+  background-color: #dddbdb;
```
Comment (deekshithmd) [inline css/components/button/index.css]: using color variab;le will be good

File: css/components/alert/index.css (hunk 1 above)
Comment (sanjitsarkar) [reply css/components/alert/index.css]: Okay.
I will make change.

File: css/components/input/index.css [hunk 6]
```diff
@@ -0,0 +1,238 @@
+@import url('../slider/index.css');
+
+input{
+    display: inline-block;
+    border: 0;
+    padding: 0.6em .7em;
+    border-radius: 0.2rem;
+    background-color: rgba(255, 255, 255, 0.034);
```
Comment (deekshithmd) [inline css/components/input/index.css]: using color variable will be good

File: css/components/input/index.css [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+    display: block;
+  }
+  
+  /* Style the checkmark/indicator */
+  .checkbox-container .checkmark:after {
+    left: .5rem;
+    top: .3rem;
+    width: .4rem;
+    height: .7rem;
+    border: solid white;
```
Comment (deekshithmd) [inline css/components/input/index.css]: use color variables

**Review by deekshithmd — APPROVED** [review]

File: css/components/avatar/index.css (hunk 2 above)
Comment (bossmonkey98) [reply css/components/avatar/index.css]: +1

File: css/components/avatar/index.css [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+  height: 5rem;
+}
+
+.avatar-sm {
+  width: 4rem;
+  height: 4rem;
+}
+
+.avatar-xsm {
+  border-width: 0.15rem;
```
Comment (bossmonkey98) [inline css/components/avatar/index.css]: Use px here

File: css/components/avatar/index.css [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+    transition: all .3s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+    box-shadow: var(--bx-sh-3); */
+}
+.avatar-badge {
+  right: 0%;
+  top: 0%;
+  position: absolute;
+  border-radius: 50%;
+  min-width: 2rem;
+  padding: 0.25rem;
```
Comment (bossmonkey98) [inline css/components/avatar/index.css]: +1

File: css/util/typography/index.css [hunk 10]
```diff
@@ -0,0 +1,196 @@
...
+}
+.font-extrabold {
+  font-weight: 800;
+}
+.font-black {
+  font-weight: 900;
+}
+
+.leading-3 {
+  line-height: 0.75rem;
```
Comment (bossmonkey98) [inline css/util/typography/index.css]: Use px for values less than 10px

**Comment by bossmonkey98** [comment]
Everything's fine just use px for values less than 10px

**Review by bossmonkey98 — APPROVED** [review]

**Review by bossmonkey98 — APPROVED** [review]

**Pull request merged**